src/commands/sign.ts

```diff
@@ -0,0 +1,85 @@
+import { Argv } from "yargs";
+import signale from "signale";
+import { sign, Output } from "../implementations/sign";
+import { isDir } from "../implementations/diskUtils";
+
+export const command = "sign <raw-documents-path>";
+
+export const describe = "Sign document(s) and appends proof block";
+
+export const builder = (yargs: Argv): Argv =>
+  yargs
+    .positional("raw-documents-path", {
+      description: "Directory containing the unissued raw documents or a single raw document file",
+      normalize: true,
+      type: "string"
+    })
+    .option("output-file", {
+      alias: "of",
+      description: "Write output to a file. Only use when <wrapped-documents-dir> is a document",
+      type: "string",
+      conflicts: "output-dir"
+    })
+    .option("output-dir", {
+      alias: "od",
+      description: "Write output to a directory",
+      type: "string",
+      conflicts: "output-file"
+    })
+    .option("private-key", {
+      alias: "private-key",
+      description: "Private key to sign document(s) with",
+      type: "string"
+    })
+    .option("public-key", {
+      alias: "public-key",
+      description: "Public key to sign document(s) with",
+      type: "string"
+    });
+
+export interface SignArguments {
```

Comment:
I would suggest to follow some commands and export this to a file so that dont need to duplicat ethe type all over the place. I let you find an example in the code ;)